physics/GFS_rrtmgp_cloud_mp.F90

```diff
@@ -875,17 +875,12 @@ subroutine cmp_reff_Thompson(nLev, nCol, i_cldliq, i_cldice, i_cldsnow, i_cldice
           qi_mp(iCol,iLay) = tracer(iCol,iLay,i_cldice)    / (1.-q_lay(iCol,iLay))
           qs_mp(iCol,iLay) = tracer(iCol,iLay,i_cldsnow)   / (1.-q_lay(iCol,iLay))
           ni_mp(iCol,iLay) = tracer(iCol,iLay,i_cldice_nc) / (1.-q_lay(iCol,iLay))
-          if (ltaerosol) then
+          if (ltaerosol .or. mraerosol) then
```

Comment:
This is fine, but probably not needed.
The RRTMGP-ThompsonMP coupling is in a state of disarray and needs to be updated with many changes that have occurred in [RRTMG](https://github.com/ufs-community/ccpp-physics/blob/244bb4365f54d9cffbe943714eee84f34b1edeca/physics/GFS_rrtmg_pre.F90#L880), but were not ported to RRTMGP. (It's on my list to do this at somepoint...)